### ISSUE TITLE
[css-typed-om-1] Does computedStyleMap() always return a new object?

### ISSUE THREAD

**Issue by bzbarsky** [w3c/css-houdini-drafts#660]
https://drafts.css-houdini.org/css-typed-om-1/#computed-stylepropertymapreadonly-objects

If so, it should be annotated [NewObject].

That said, why would it?  There's no reason for it too, afaict, just like attributeStyleMap and styleMap don't.

**Comment by darrnshn** [comment]
Blink currently returns a new object each time. I'm not sure what the reason is, my guess is maybe match `getComputedStyle()`?

**Comment by bzbarsky** [comment]
Right, but for getComputedStyle() it's sort of a function of window and element, so returning  new thing each time sort of makes sense because caching it would be a bit weird and complicated.  In this case we just have an element.

**Comment by tabatkins** [comment]
Yeah, we should be able to return the same object each time, I think.  It's explicitly live, so the get() calls will always reach down to the real element anyway.  I guess that means it should use a `[SameObject]` annotation?

**Comment by bzbarsky** [comment]
That would work.

**Comment by bzbarsky** [comment]
(Same object every time means it needs to get cached on the element somewhere, so there's that tradeoff.)

**Comment by tabatkins** [comment]
Ah, I don't actually use `[SameObject]` - that's for attributes. I just need to specify in the method that it caches on first access.
